pyproject.toml

```
@@ -53,7 +53,7 @@ types = [
 docs = [
     "canonical-sphinx[full]",
     "sphinx-lint",
-    "sphinx-autobuild<2024.2.5" # 2024.02.04 drops support for Python 3.8
+    "sphinx-autobuild<2024.9.20" # 2024.02.04 drops support for Python 3.8
```

```suggestion
    "sphinx-autobuild"
```

Since we're presumably autobuilding with 3.10+